### ISSUE TITLE
RuntimeWarning on package load

### ISSUE THREAD

**Issue by gunnarvoet** [wesleybowman/UTide#124]
Awesome package - I really enjoy using this a lot!

With python 3.11 and numpy 1.26.4 I get the following `RuntimeWarning`s when loading the package:
```
utide/harmonics.py:16: RuntimeWarning: invalid value encountered in cast
  nshallow = np.ma.masked_invalid(const.nshallow).astype(int)
utide/harmonics.py:17: RuntimeWarning: invalid value encountered in cast
  ishallow = np.ma.masked_invalid(const.ishallow).astype(int) - 1
```

This can be fixed by moving the cast to `int` a few lines down where `nshallow` and `ishallow` are compressed. Would you like me to submit this fix as a PR?

**Comment by efiring** [comment]
Yes, please do so.  This problem is a consequence of the old Matlab heritage.  It could be fixed more cleanly, but with quite a bit more disruption, risk, and effort.  I think your suggested quick fix is good for now, and well worth doing.

**Comment by gunnarvoet** [comment]
Thanks @efiring !

**Comment by efiring** [comment]
Closed by #125.  Thank you, @gunnarvoet.
